.github/workflows/testing.yml

```diff
@@ -231,3 +231,78 @@ jobs:
           python -m pip install --upgrade .
       - name: Test PDF generation on Windows
         run: pytest test/test_output_engine.py -k test_output_pdf
+
+  windows_long_tests:
+    name: Windows long tests
+    runs-on: windows-2019
+    timeout-minutes: 120
+    env:
+      LONG_TESTS: 1
+      NO_EXIT_CVE_NUM: 1
+      PYTHONIOENCODING: 'utf8'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          cache: 'pip'
+      - name: Get date
+        id: get-date
+        run: |
+          echo "::set-output name=date::$(get-date -format "yyyyMMdd")"
+      - name: Get cached database
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/cve-bin-tool
+          key: ${{ runner.os }}-cve-bin-tool-${{ steps.get-date.outputs.date }}
+      - name: Install cve-bin-tool
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade setuptools
+          python -m pip install --upgrade wheel
+          python -m pip install --upgrade .
+      - name: Try single CLI run of tool
+        run: |
+          python -m cve_bin_tool.cli test/assets/test-kerberos-5-1.15.1.out -n json -u latest
+      - name: Run async tests
+        run: >
+          pytest --cov --cov-append -n 4 -v
+          --ignore=test/test_cli.py
+          --ignore=test/test_cvedb.py
+      - name: Run synchronous tests
+        run: >
+          pytest -v --cov --cov-append --cov-report=xml
+          test/test_cli.py
+          test/test_cvedb.py
+      - name: Cache conda
+        uses: actions/cache@v3
+        env:
+          # Increase to reset cache if requirements.txt file has not changed
+          CACHE_NUMBER: 0
+        with:
+          path: ~/conda_pkgs_dir
+          key:
+            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
+            hashFiles('requirements.txt') }}
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          activate-environment: pdftotext
+          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
+      - name: Install pdftotext, reportlab and cve-bin-tool
+        run: |
+          conda install -c conda-forge python=${{ matrix.python }} poppler pdftotext
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade setuptools
+          python -m pip install --upgrade wheel
+          python -m pip install --upgrade reportlab
+          python -m pip install --upgrade .
+      - name: Test PDF generation on Windows
+        run: pytest test/test_output_engine.py -k test_output_pdf --cov --cov-append --cov-report=xml
+      - name: Upload code coverage to codecov
+        uses: codecov/codecov-action@v3
+        with:
+          files: ./coverage.xml
+          flags: win-longtests
+          name: codecov-umbrella
+          fail_ci_if_error: false
```

Comment:
It's a good practice to add a newline at the end of a file.
Relevant discussion: https://unix.stackexchange.com/a/18789

Reply:
Hah, that's exactly the problem we just had with the spell checker, where it didn't parse the last entry in the file because there was no newline. So yes, newlines at end of files are a good habit to get into!

Reply:
Alright thanks!!